mesonpy/__init__.py

```diff
@@ -82,6 +73,16 @@ class _depstr:
     'reset': '\33[0m',
 }
 _NO_COLORS = {color: '' for color in _COLORS}
+_NINJA_REQUIRED_VERSION = (1, 8, 2)
+
+
+class _depstr:
+    """Namespace that holds the requirement strings for dependencies we *might*
+    need at runtime. Having them in one place makes it easier to update.
+    """
+    patchelf = 'patchelf >= 0.11.0'
+    wheel = 'wheel >= 0.36.0'  # noqa: F811
+    ninja = f'ninja >= {".".join(_NINJA_REQUIRED_VERSION)}'
```

Comment:
You took my code snipped too seriusly: this is wrong 😄 You need something like:
```python
    ninja = f'ninja >= {".".join(str(i) for i in _NINJA_REQUIRED_VERSION)}'
```

Reply:
Ahhh!